test/webpack-integration.test.js

```diff
@@ -37,6 +40,59 @@ describe('Webpack Integration Tests', () => {
       });
     });
   });
+
+  it('calls cssProcessor with correct arguments', (done) => {
+    fs.readFile(__dirname + '/default/default.css', (err, data) => {
```

Comment:
these tests are a little hard to reason about. A good start might be to use `readFileSync` instead of `readFile` so you don't need callback hell like this...

Reply:
Yeah Good Call